src/widgets/common.tsx

```diff
@@ -40,6 +40,7 @@ export const ChartAndMetricsWrapper = styled.div`
   display: flex;
   flex-direction: column;
   flex: 1;
+  margin-bottom: 70px;
```

Comment:
This line looks problematic. The `ChartAndMetricsWrapper` is a widely used component. Is it a good idea to set such a fixed margin here?